lib/mpl_toolkits/axes_grid1/axes_divider.py

```diff
@@ -299,16 +328,8 @@ def __init__(self, axes_divider, nx, ny, nx1=None, ny1=None):
         self._nx, self._ny = nx - _xrefindex, ny - _yrefindex
 
         if nx1 is None:
-            _api.warn_deprecated(
-                "3.5", message="Support for passing nx1=None to mean nx+1 is "
-                "deprecated since %(since)s; in a future version, nx1=None "
-                "will mean 'up to the last cell'.")
```

Comment:
Maybe a removal note for this? (And similar changes.)

Or simply keep them as the whole class is deprecated anyway?

Reply:
It is slightly easier to remove them because HVBoxDivider._locate is still used internally.  From the PoV of the end user I think it's fine because they can't publically access these (with values that trigger deprecations).